indexing-service/src/test/java/io/druid/indexing/overlord/TestRemoteTaskRunnerConfig.java

```diff
@@ -33,7 +33,8 @@ public Period getTaskAssignmentTimeout()
   @Override
   public long getMaxZnodeBytes()
   {
-    return 1000;
+    // make sure this is large enough, otherwise RemoteTaskRunnerTest might fail unexpectedly
```

Comment:
no, I only saw it randomly pop up once in the logs. I think it may be due to the way tasks are spawned off, which causes the errors to not show up, but I'm not sure.